modules/persistent-storage-csi-gcp-pd-storage-class-ref.adoc

```diff
@@ -14,7 +14,9 @@ The GCP PD CSI driver uses the `csi.storage.k8s.io/fstype` parameter key to supp
 |===
 |Parameter  |Values  |Default  |Description
 
-|`type` | `pd-ssd` or `pd-standard` | `pd-standard` | Allows you to choose between standard PVs or solid-state-drive PVs.
+|`type` | `pd-ssd`, `pd-standard`, or `pd-balanced` | `pd-standard` | Allows you to choose between standard PVs or solid-state-drive PVs.
+
+*Note*: The driver does not validate the value, thus all the possible values are accepted.
```

Comment:
Can we add "proper" notes to table cells?

Reply:
Just decided to leave out the "note" aspect.